### ISSUE TITLE
The sample app is missing from play store. 

### ISSUE THREAD

**Issue by irshadshalu** [divyanshub024/AndroidDraw#28]
The sample app url given in README.md is not available in play store anymore. Please consider adding the release APK here as a release and Linking it.

**Comment by divyanshub024** [comment]
Yeah, play store removed the app due some issues. I'll fix that and re-upload the app in the coming week.

**Comment by irshadshalu** [comment]
Yeah, Google does this every now and then. It will be nice if you could put an APK in github releasw itself in case this happens again

**Comment by divyanshub024** [comment]
Sure, I'll do that.
